tests/integration/dlc/dlc.test.ts

```diff
@@ -222,26 +224,22 @@ describe('dlc provider', () => {
       });
 
       it('close', async () => {
```

Comment:
We should probably add a test for passing in fixedInputs to `createDlcClose`

Reply:
Would be good to have a test for error cases, such as finalizeDlcClose with dlcClose that has invalid `fundingInputs` or invalid `closeSignature`

Reply:
Good idea, updated tests to reflect that